install/operator/pkg/cmd/internal/install/install.go

```diff
@@ -99,6 +99,7 @@ func New(parent *internal.Options) *cobra.Command {
 		},
 	}
 	app.PersistentFlags().StringVarP(&o.addons, "addons", "", "", "a comma separated list of addons that should be enabled")
```

Comment:
Can't we remove the `addons` option now?

Reply:
Probably. I'll put it in on the TODO list.